client/flutter/lib/api/user_preferences.dart

```diff
@@ -40,6 +40,17 @@ class UserPreferences {
         .setBool(UserPreferenceKey.AnalyticsEnabled.toString(), value);
   }
 
+  Future<bool> getNotificationsEnabled() async {
```

Comment:
are these being used anywhere?

Reply:
these look right... will be used if not already